Sources/ScipioKit/Producer/PIF/PIFGenerator.swift

```diff
@@ -0,0 +1,140 @@
+            for baseTarget in project.targets {
+                if let pifTarget = baseTarget as? PIF.Target {
+                    let isObjectTarget = pifTarget.productType == .objectFile
+
+                    guard [.objectFile, .bundle].contains(pifTarget.productType) else { continue }
+
+                    let name = pifTarget.name
+                    let c99Name = pifTarget.name.spm_mangledToC99ExtendedIdentifier()
+
+                    if isObjectTarget {
+                        pifTarget.productType = .framework
+                        pifTarget.productName = "\(name).framework"
+                    }
+
+                    let newConfigurations = pifTarget.buildConfigurations.map { original in
+                        var configuration = original
+                        var settings = configuration.buildSettings
+
+                        if isObjectTarget {
+                            settings[.PRODUCT_NAME] = "$(EXECUTABLE_NAME:c99extidentifier)"
+                            settings[.PRODUCT_MODULE_NAME] = "$(EXECUTABLE_NAME:c99extidentifier)"
+                            settings[.EXECUTABLE_NAME] = c99Name
+                            settings[.TARGET_NAME] = name
+                            settings[.PRODUCT_BUNDLE_IDENTIFIER] = name.spm_mangledToBundleIdentifier()
+                            settings[.CLANG_ENABLE_MODULES] = "YES"
+                            settings[.DEFINES_MODULE] = "YES"
+                            settings[.SKIP_INSTALL] = "NO"
+                            settings[.INSTALL_PATH] = "/usr/local/lib"
+                            settings[.ONLY_ACTIVE_ARCH] = "NO"
+
+                            switch buildOptions.frameworkType {
+                            case .dynamic:
+                                settings[.MACH_O_TYPE] = "mh_dylib"
+                            case .static:
+                                settings[.MACH_O_TYPE] = "staticlib"
+                            }
+
+                            settings[.LIBRARY_SEARCH_PATHS, default: ["$(inherited)"]]
+                                .append("\(buildParameters.toolchainLibDir.pathString)/swift/\(sdk.settingValue)")
+
+                            settings[.GENERATE_INFOPLIST_FILE] = "YES"
+
+                            // Set the project and marketing version for the framework because the app store requires these to be
+                            // present. The AppStore requires bumping the project version when ingesting new builds but that's for
+                            // top-level apps and not frameworks embedded inside it.
+                            settings[.MARKETING_VERSION] = "1.0" // Version
+                            settings[.CURRENT_PROJECT_VERSION] = "1" // Build
+
+                            // Enable `-enable-library-evolution` to emit swiftinterface
+                            settings[.OTHER_SWIFT_FLAGS, default: ["$(inherited)"]]
+                                .append("-enable-library-evolution")
```

Comment:
Some libraries don't support `library-evolution` like SwiftNIO.
So I'll add the option to specify this by each framework.